airbyte-integrations/connectors/source-iterable/source_iterable/schemas/campaigns_metrics.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
```

Comment:
is there is a reason why we have so generic schema?

Reply:
As I can see there's no a strict schema for this

Reply:
I don't think this can be useful for users, because it lacks campaign id, also we definitely know at least some structure:
metric: <string>
value: <number>

Reply:
the id of campaign is inside of `data`, lets maybe rename it to metrics? or even better move it to the top level of object? i.e. `campaign_metrics` will looks like:
```json
{
<id>: <campaign_id>,
<metric_i>: <value_i>
}
```